google/cloud/bigtable/data/_async/client.py

```diff
@@ -483,35 +501,31 @@ async def read_rows_stream(
             - GoogleAPIError: raised if the request encounters an unrecoverable error
             - IdleTimeout: if iterator was abandoned
         """
+        operation_timeout = (
+            operation_timeout or self.default_read_rows_operation_timeout
+        )
+        attempt_timeout = (
+            attempt_timeout
+            or self.default_read_rows_attempt_timeout
+            or operation_timeout
```

Comment:
I dont think we want to fall back on operation_time for read_rows or mutate_rows

Reply:
This is the final operation_timeout for the call, configured one line above, not the table's default_operation_out.

The purpose of this is to set the attempt_timeout == the operation_timeout if the attempt timeout remains unset, so we don't have to do None checks later on